CI/Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN apt-get -y update; \
                        python3 \
                        python3-pip \
                        python3-setuptools \
+                       python3-packaging \
```

Comment:
```suggestion
```
shall we remove this and retrigger the CI

Reply:
🤞🏻